### ISSUE TITLE
iPQF: spectra/PSMs or peptides

### ISSUE THREAD

**Issue by lgatto** [lgatto/MSnbase#55]
@martifis In you comments, you mention that you start from spectra/PSMs to summarise quantitative data into proteins. I believe you also use mzTab data as input (which is really great, by the way). I am working on an updated mzTab importer (see `MzTab`, but that's only the first step; eventually, this will produce `MSnSet` instaces). While reading the specifications for the `mzTab` format version 1.0 (updated 20 June 2014), I understand that there is not quantitation data at the spectra/PSM level, but only at the peptides (or protein) level; more specifically, in the PSM section on page 12-13, table 5, I don't see any abundance field. Could you refer to this, or any other relevant specifications, so that I make sure that the new `mzTab` importer and iPQF work well together?

**Comment by martifis** [comment]
My original test data sets were no mzTAB data, but spectra-level outputs from Mascot and ProteinPilot - I transferred these later into a MSnSet. My focus was that iPQF can run based on a MSnSet object: 1) concerning the abundances I then relied on abundances within the constructed MSnSet and I only call it by 'exprs(MSnSet object)' within iPQF. 2) However, concerning feature names of the fData(MSnSet) I need to have fixed feature colnames I can refer to in my code. Here I chose mzTab names given in the PSM Section (page 12/13): "sequence", "accession", "charge", "modifications", "search_engine_score", "mass_to_charge" ( I just saw in the PSM section this is called "calc_mass_to_charge"). 
Hence here, I circumvented the problem and did not face the problem of reading in PSM data from a mzTab file. However, I was also wondering why mzTab specifications do not list abundance information in the PSM section - maybe does mzTAB  not intend to provide abundances on spectra level?
